src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  onDelete: (id: number) => void;
+  todoId: number | null;
+  setTodoId: (id: number | null) => void;
+}
+
+export const TodoItem: FC<Props> = ({
+  todo,
+  onDelete,
+  todoId,
+  setTodoId,
+}) => {
+  const { id, title, completed } = todo;
+
+  const handleDelete = (selectedTodoId: number) => {
```

Comment:
handleDelete function does not need to have 'selectedTodoId' as a parameter, as it can access 'id' directly from the 'todo' prop.